content/en/blog/_posts/2024-11-11-memory-manager-moves-to-ga/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Memory Manager Goes GA"
+date: 2024-11-11
+slug: memory-manager-goes-ga
+author: >
+  [Talor Itzhak](https://github.com/Tal-or) (Red Hat)
+draft: true
+---
+
+With Kubernetes 1.32, the memory manager has officially graduated to General Availability (GA),
+marking a significant milestone in the journey toward efficient and predictable memory allocation for containerized applications.
+Since Kubernetes v1.22, where it graduated to beta, the memory manager has proved itself reliable, stable and a good complementary feature for the
+[CPU Manager](/docs/tasks/administer-cluster/cpu-management-policies/).
+
+As part of kubelet's workload admission process, 
+the memory manager provides topology hints 
+to optimize memory allocation and alignment. 
+This enables users to allocate exclusive
+memory for `Guaranteed` [QoS class](/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes) pods.
```

Comment:
One nit
```suggestion
memory for Pods in the [Guaranteed](/docs/concepts/workloads/pods/pod-qos/#guaranteed) QoS class.
```

We should link to the concept not to a task page.

Reply:
Done. Thanks!